src/ontology/mondo-edit.obo

```diff
@@ -584370,6 +584370,17 @@ is_a: MONDO:1030005 {source="PMID:31120550"} ! mosaic SMO syndrome
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7768" xsd:anyURI
 
+[Term]
+id: MONDO:1030008
+name: mitral valve insufficiency
+def: "Dysfunction of the mitral valve characterized by incomplete valve closure." [https://orcid.org/0009-0009-6791-365X, NCIT:C50888]
```

Comment:
please update the definition to make it clear that this term refers to a disease/disorder. 
For example: "A mitral valve disorder characterized by (blablabla)"

Reply:
updated

Reply:
updated